src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -648,6 +648,33 @@
   <data name="GeneralPage_CheckForUpdates.Content" xml:space="preserve">
     <value>Check for updates</value>
   </data>
+  <data name="General_SettingsBackupAndRestoreLocationText.Description" xml:space="preserve">
+    <value />
+  </data>
+  <data name="General_SettingsBackupAndRestoreLocationText.Header" xml:space="preserve">
+    <value>Location</value>
+  </data>
+  <data name="General_SettingsBackupAndRestoreStatusInfo.Description" xml:space="preserve">
+    <value />
+  </data>
```

Comment:
why is this empty? @jefflord did you changed something? If not needed please remove it.

Reply:
There was commented out data in there but the UI removed it. Which sucks.

Reply:
We might want that back, UI still in flux

Reply:
If we want to have it back then we can recreate it. Empty entries are a bad idea.

Reply:
Correct.